Users/Docker_Image.rst

```diff
@@ -1,3 +1,47 @@
+What is Docker?
+===============
+
+Docker is a tool designed to make it easier to create, deploy, and run
+applications by using containers. Containers allow a developer to package
+up an application with all of the parts it needs, like libraries and other
+dependencies, and ship it all out as a single package.
+
+With the help of containers, the developer can rest assured that
+the application will run on other machines regardless of any
+customized settings his machine might have that could differ from the
+machine used for writing and testing the code.
+
+Unlike a virtual machine, rather than creating a whole virtual operating
+system, Docker allows applications to use the same kernel as the system
+that they're running on and only requires applications being shipped
+with things not already running on the host computer. This gives a
+significant performance boost and reduces the size of the application.
+
+What is a Docker Image and how is it different from a container?
+----------------------------------------------------------------
+
+An image is an immutable file which contains required binaries and libraries
+needed to make a container and a running instance of an image is called
+a container. Images are composed of layers of other images. Images are
+created when we run the ``build`` command of Docker and containers are formed
+from these images when we use the ``run`` command of Docker. There can be
+many containers for the same image.
+
+For more information about Docker see the
+`official documentation <https://docs.docker.com/>`__.
+
+Installing Docker
+-----------------
+
+Docker installation guide for various operating systems can be found in the
+`official Docker installation instructions <https://docs.docker.com/engine/installation/>`__.
+
+.. note::
+
+  Downloading Docker images and pushing images over low bandwidth connections
+  is very slow.
```

Comment:
What about
> Docker images are usually very large. Downloading or pushing them over low bandwidth connections can be very slow.

instead? :)
your choice :+1:

Reply:
Agree with you. Done necessary changes